bitmind/utils/mock.py

```diff
@@ -1,10 +1,92 @@
 import time
-
 import asyncio
 import random
 import bittensor as bt
-
+import numpy as np
 from typing import List
+from PIL import Image
+
+from bitmind.constants import DIFFUSER_NAMES
+
+
+def create_random_image():
+    random_data = np.random.randint(0, 256, (512, 512, 3), dtype=np.uint8)
+    return Image.fromarray(random_data)
+
+
+class MockImageDataset:
```

Comment:
[Recommendation] Have a mock non-comptabile synapse (something like HealthI) to test the negative case